src/components/UserProfile/UserProfileHeader/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { withStyles } from "@material-ui/styles";
+
+import UserProfileCard from "../../common/UserProfileCard";
+import { useStyles } from "./styles";
+
+const UserProfileHeader = ({ classes, userName }) => {
+  return (
+    <div className={classes.userProfileHeader}>
+      <UserProfileCard userName={userName} />
+      <Link className={classes.requestText}>request developer's account</Link>
```

Comment:
Should this be a button instead?

Reply:
[done](https://github.com/singnet/refactored-dapp/pull/17/commits/01515ec94af701617ec12dcff05ae6f7acde21e4)